{
  try {
    finallyHandler.accept(ctx,command,returnValue,null);
    return this;
  }
 catch (  Throwable t) {
    return new ExceptionStage(ctx,command,t);
  }
}
